src/guiVolumeChange.cpp

```diff
@@ -53,22 +52,22 @@ void GUIVolumeChange::removeChildren()
 {
 	{
 		gui::IGUIElement *e = getElementFromId(ID_soundText1);
-		if(e != NULL)
+		if (e != NULL)
```

Comment:
in fact if you want to cleanup the code there, replace 

{
gui::IGUIElement *e = getElementFromId(ID_soundText1);
if (e != NULL)
e->remove()
}

with

if (gui::IGUIElement *e = getElementFromId(ID_soundText1))
    e->remove();